05LessonFive/Checkpoint1/css/style.css

```diff
@@ -1,43 +1,32 @@
-.lawn {
-    width: 100%;
-    height: 300px;
-}
+img{
 
-.lawn h1 {
-    text-align: center;
-    font-family: monospace;
-    color: darkgray;
 }
-
-.lawn {
-    background-image: url(../img/banner-garden-stmichaelsmount.jpg);
-    background-repeat: no-repeat;
-    background-position: center;
-    background-size: cover;
+h2{
+  color:#252b3b;
+font-size: 1.9em;
 }
-
-.left {
-    float: left;
-    width: 60%;
-    height: 300px;
-    margin-left: 260px;
-    background-color: lightgray;
+p{
+  color:#555;
+  padding-bottom:55px;
 }
+h3{
+  border:2px solid #555;
+  padding-top:5px;
+  padding-right:5px;
+  padding:bottom:5px;
+  padding:left:5px;
+  color:#252b3b;
+margin:10px;
```

Comment:
Inside your h3 CSS, change:

margin: auto;
width: 100px;

to fix your link issue.